__tests__/fixtures/install/missing-opt-peer-dep/foo/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "foo",
+  "version": "1.0.0",
+  "peerDependencies": {
+    "bar": "optional:*"
```

Comment:
Again, this will break downstream consumers and people using older versions of yarn, and any version of npm. It will mean that every airbnb library can not depend on any library using this syntax. I strongly encourage you to reconsider this hugely ecosystem-breaking approach.

Reply:
Again, that's not the case. Please refer to [my chart](https://www.npmjs.com/package/semver) that explains how all package managers will support this syntax with a graceful degradation. If you have additional questions, feel free to ping me for a direct meeting, I think I've explained as much as I could by text.

> and any version of npm

Then I suggest you push them to collaborate. I can't do much more than I already did, and if they wish to ghost us there's little I personally can do to stop them. I agree it's damaging to the ecosystem, but maybe it should have been considered before giving them the keys to the city.

Reply:
For install, yes. But `npm ls` will break.

Reply:
“Doing nothing” is far better than breaking things; so I’m not sure why you have to proceed rapidly when there’s no better option yet available.

Reply:
Fixed by 7d61b4f800fb4150078b0726a34b4e3c09670dfe